### PR TITLE
Development-specific tasks are not part of the default install task any more

### PR DESCRIPTION
**Pull request by armish**
With this, people have to

```
npm install
npm run build  # or npm run prod
```

for jstransform, browserify and uglify tasks.

[Review on Reviewable](https://reviewable.io/reviews/hammerlab/pileup.js/213)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,6 +67,7 @@ To style multiple tracks of the same type, you can use the `cssClass` property.
     git clone https://github.com/hammerlab/pileup.js.git
     cd pileup.js
     npm install
+    npm run prod
```
Comment (danvk) [inline README.md]: How about `npm run build` and drop the `prod` alias?

Comment (armish) [inline README.md]: Sure -- I actually added it there because I thought you liked the `prod` tasks :)

Comment (danvk) [inline README.md]: Nah, one task is fine :)

**Comment by danvk** [comment]
LGTM, feel free to merge on green

**Pull request merged**